packages/eyes/src/generate-snapshots/map-styles-to-components.ts

```diff
@@ -0,0 +1,15 @@
+import path from 'path';
+import {IRegistry} from '@ui-autotools/registry';
+
+export function mapSylesToComponents(Registry: IRegistry, projectDirectory: string): {[stylePath: string]: string} {
+  const mapping: {[stylePath: string]: string} = {};
+
+  Registry.metadata.components.forEach((componentMetadata) => {
+    if (componentMetadata.exportName) {
+      // We map styles to their components here (i.e. blah/blah/style1.st.css -> blah/blah/myComp.tsx)
```

Comment:
This comment makes sense as the description of the entire function. Also it shows relative paths in the example, which is slightly confusing because we're mapping absolute paths.

```
Map paths of base stylesheets to paths of respective components, e.g.:
/my-proj/src/checkbox.st.css -> /my-proj/src/checkbox.tsx
```